charts/incubator/minecraft-bungeecord/values.yaml

```diff
@@ -41,7 +41,6 @@ service:
       main:
         port: 25565
         targetPort: 25577
-        protocol: HTTP
```

Comment:
AFAIK, minecraft is a TCP port.
If it was HTTP it would be able to use ingress :D

Reply:
i missed that my bad, im positive i copied this from mc-java

Reply:
i didnt infact. next  time ill just copy it and carefully read it over.